src/include/ipc4/base_fw.h

```diff
@@ -44,17 +44,6 @@
 /* Max pipeline count supported by ipc4 */
 #define IPC4_MAX_PPL_COUNT 16
 
-enum ipc4_basefw_properties {
-	IPC4_FW_VERSION,	/**< Firmware version */
-	IPC4_DSP_CORES,	/**< dsp cores count in audio subsystem */
-	IPC4_MEM_PAGE_SIZE,	/**< memory page size */
-	IPC4_TOTAL_PHYS_MEM_PAGES,	/**< total memory in page size */
-	IPC4_DL_MAILBOX_SIZE,	/**< download mailbox size */
-	IPC4_UL_MAILBOX_SIZE,	/**< uplaod mailbox size */
-	IPC4_TRACE_LOG_SIZE,	/**< trace log buffer size */
```

Comment:
Hmm, but the SOF driver _IS_ using these properties: https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/ipc4-loader.c#L203 .. there's some mis-alignment here. @ujfalusi @mwasko @lgirdwood

Reply:
Apart from the log size, we are also querying max libs count for example.

Reply:
@dnikodem  @mwasko @ujfalusi this was a false alarm. Host uses IPC4_TRACE_LOG_BYTES_FW_CFG of FW_CONFIG, which is confusingly similarly named. But, but driver uses the FW_CONFIG definitions and queries the data via FW_CONFIG for all pieces of data I could find.